backend/Dockerfile

```diff
@@ -19,8 +19,7 @@ RUN apk upgrade --no-cache --available \
 ENV CHROME_BIN=/usr/bin/chromium-browser \
     CHROME_PATH=/usr/lib/chromium/ \
     PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium-browser \
-    PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true \
-    GHOSTSCRIPT_PATH=/usr/bin/gs
```

Comment:
It looks like this is still used in application.js?

Reply:
@weskubo-cgi It sure is.  The default folder for `compress-pdf` is already `/user/bin/gs`.  I realized that the option key has changed upstream.  I put this variable back and updated the option key instead.